example/src/main/java/com/github/appintro/example/ui/java/JavaIntro.java

```diff
@@ -56,9 +56,6 @@ public void onCreate(@Nullable Bundle savedInstanceState) {
         //Activate wizard mode (Some aesthetic changes)
         setWizardMode(true);
         
-        //Show/hide skip button
-        setSkipButtonEnabled(true);
```

Comment:
I've removed this as it was clashing with `setWizardMode(true)` resulting in having both skip and the arrow on top of each other